Content.Server/Wires/WiresSystem.cs

```diff
@@ -664,7 +663,9 @@ public void SetWiresPanelSecurityData(EntityUid uid, WiresPanelComponent compone
 
         component.WiresAccessible = wiresPanelSecurityLevelPrototype.WiresAccessible;
         component.WiresPanelSecurityExamination = wiresPanelSecurityLevelPrototype.Examine;
-        Dirty(component);
+        component.WeldingAllowed = wiresPanelSecurityLevelPrototype.WeldingAllowed;
+
+        Dirty(uid, component);
 
```

Comment:
No need to copy prototype data, just get the prototype itself.

Reply:
Done